CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+The most noteworthy change of this release is the update of the container's base image from Debian 11 ("Bullseye") to Debian 12 ("Bookworm"). This update alone involves breaking changes and requires a careful update!
+
+### Breaking
+
+- **Updated base image to Debian 12**
+- Changed the default of `DOVECOT_COMMUNITY_REPO` to `0` (disabled) - the Dovecot community repo will (for now) not be the default when building the DMS.
+- Updated packages. For an overview, [we have a review comment on the PR that introduces Debian 12](https://github.com/docker-mailserver/docker-mailserver/pull/3403#issuecomment-1694563615)
+  - Notable major version bump: `openssl 3`, `clamav 1`, `spamassassin 4`, `redis-server 7`.
+  - Notable minor version bump: `postfix 3.5.18 => 3.7.6`
```

Comment:
rspamd minor version probably got bumped too, or at least consistent if you want to include that here in the changelog for visibility.

Reply:
I think we already shop 3.7.5 on our `:edge` builds TBH, which is AFAIK the very latest version of Rspamd.

Reply:
> Rspamd will not see a version bump, so no note necessary.

~~Won't this be the first release where rspamd has the same version on AMD64 and ARM64?~~ (_**EDIT:** Nope that'd be 13.1 now_)

The [v13.1 PR removes from rspamd docs](https://github.com/docker-mailserver/docker-mailserver/pull/3686/files#diff-d6195d723286b5393e85a1f0f2127d1f72a1c814387f1f1b3f10b9eee85f98ddL13) stating rspamd 3.5 (AMD64) and 3.4 (ARM64).

Granted, since you've chosen to continue with a v13.1 release there won't be a version change to announce here, but it's [not mentioned explicitly in that PR changelog entry either](https://github.com/docker-mailserver/docker-mailserver/pull/3686/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR28).

Reply:
We can add this to the release PR of the v13.1.0 release.